### PR TITLE
Create moviejs.yml

### PR DESCRIPTION
**Pull request by clabe45**

### CODE REVIEW

File: _data/projects/moviejs.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+site: https://github.com/clabe45/movie.js
+tags:
+- javascript
+- vanilla-javascript
+- video
+- video-editing
+- automation
+upforgrabs:
+  name: help wanted
+  link: https://github.com/clabe45/movie.js/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22
```
Comment (ritwik12) [inline _data/projects/moviejs.yml]: @clabe45 Please change this to https://github.com/clabe45/movie.js/labels/help%20wanted

**Review by ritwik12 — CHANGES_REQUESTED** [review]

File: _data/projects/moviejs.yml (hunk 1 above)
Comment (clabe45) [reply _data/projects/moviejs.yml]: Ok

Comment (clabe45) [reply _data/projects/moviejs.yml]: It's updated now

**Review by ritwik12 — APPROVED** [review]

**Comment by ritwik12** [comment]
@clabe45 Sorry to keep you waiting.

**Comment by clabe45** [comment]
No problem thanks

**Pull request merged**